autoextract/aio/client.py

```diff
@@ -147,6 +280,8 @@ def request_parallel_as_completed(query: Query,
                                   batch_size=1,
                                   n_conn=1,
                                   agg_stats: AggStats = None,
+                                  handle_retries=True,
```

Comment:
@victor-torres sorry, I missed this - what's the use case for adding this argument to request_parallel_as_completed? When False is passed, won't a single throttling error cause an exception, and stop the processing?

Reply:
Just keeping interfaces consistent.

Reply:
Should I keep it? Let me know and I can remove it in another PR.

Reply:
I think it makes sense to remove it - it'd be a new feature, with unclear use case, which can be actually hamful for users (they can turn set it to False without understanding the consequences, and then their crawl stops in the middle, after half an hour).